src/Optimizely/DecisionService/DecisionService.php

```diff
@@ -98,7 +98,7 @@ private function getBucketingId($userId, $userAttributes)
         // place of the userID for the murmur hash key
         $bucketingIdKey = ControlAttributes::BUCKETING_ID;
         
-        if (!empty($userAttributes[$bucketingIdKey])) {
+        if (!empty($userAttributes[$bucketingIdKey]) && is_string($userAttributes[$bucketingIdKey])) {
```

Comment:
See Python PR. We should be logging the case when bucketingId is not a string and we default to userID. 
You may remove the existing log messsage 'Setting the bucketing ID to "%s".